src/60_Samples_%26_Templates/Hotel.html

```diff
@@ -17,279 +18,149 @@
+<script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
 <!--
-  The `amp-access` element checks the user's current session. The JSON response includes the user's loyalty status and how many free nights they have earned.
-  ```json
-  {
-    "username":"test-user",
-    "status":"Gold",
-    "freenights":2
-  }
-  ```
+  We use `amp-list` to show a list of items.
 -->
-<div>
-    <section amp-access="status = 'Gold'">
-        <template amp-access-template type="amp-mustache">
-            <div class="loyalty-message">Congratulations <strong>{{username}}</strong>, your PAMPered <strong>{{status}}</strong> status has earned you <strong>{{freenights}}</strong> free nights you can apply to this stay.</div>
-        </template>
-    </section>
-</div>
-<!-- ## Dates of stay -->
+<script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
 <!--
-  An amp-form is used to allow the user to change the dates of their stay. The `type="date"` is used to allow the browser to display a native date picker. Learn more about handling the result of a form submission in the [form example](/components/amp-form).
+  We use `amp-bind` to bind the value of the room list to a variable that can change dinamically.
 -->
-<div>
-  <h6>Your stay</h6>
-  <p>7 nights, from 12 Dec 2020 to 06 Jan 2021</p>
-  <form method="GET" class="px3 mb4"
-    action="/samples_templates/hotel/check-available"
-    action-xhr="/samples_templates/hotel/check-available"
-    target="_top">
-    <fieldset id="stay-input" class="border-none p0 m0">
-    <div class="ampstart-input relative m0 p0 mb3">
-      <label for="checkin">Check-in:</label>
-      <input name="checkin" id="checkin" type="date" class="block border-none p0 m0" value="2020-12-30">
-    </div>
-    <div class="ampstart-input relative m0 p0 mb3">
-      <label for="checkout">Check-out:</label>
-      <input name="checkout" type="date" class="block border-none p0 m0" value="2021-01-06">
-    </div>
-    <div class="ampstart-input relative m0 p0 mb3 ">
-      <label for="guests">Guests:</label>
-      <select name="guests" class="block border-none p0 m0">
-        <option value="1">1</option>
-        <option value="2" selected>2</option>
-        <option value="3">3</option>
-      </select>
-    </div>
-    </fieldset>
-    <input type="submit" value="Check availability" class="ampstart-btn caps">
-    <div submit-success>
-      <template type="amp-mustache">
-        Yay!
-      </template>
-    </div>
-    <div submit-error>
-      <template type="amp-mustache">
-        No rooms available.
-      </template>
-    </div>
-  </form>
-</div>
-<!-- ## Available rooms -->
+<script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
 <!--
-  Another `amp-form` around the table allows for booking buttons.
+  We use `amp-date-picker` to implement a filter based on date selection.
```

Comment:
suggestion: We use `amp-date-picker` to simplify entering dates for checking room availability.

Reply:
done